sdk/src/rest/commands/update/items.ts

```diff
@@ -7,74 +7,45 @@ export type UpdateItemOutput<
 	Collection extends keyof Schema,
 	TQuery extends Query<Schema, Schema[Collection]>,
 > = ApplyQueryFields<Schema, CollectionType<Schema, Collection>, TQuery['fields']>;
-
 /**
  * Update multiple items at the same time.
  *
  * @param collection The collection of the items
- * @param keys The primary key of the items
+ * @param keysOrQuery The primary key of the items
  * @param item The item data to update
  * @param query Optional return data query
  *
  * @returns Returns the item objects for the updated items.
- * @throws Will throw if keys is empty
+ * @throws Will throw if keysorQuery is empty
  * @throws Will throw if collection is empty
  * @throws Will throw if collection is a core collection
  */
 export const updateItems =
 	<Schema extends object, Collection extends keyof Schema, const TQuery extends Query<Schema, Schema[Collection]>>(
 		collection: Collection,
-		keys: string[] | number[],
+		keysOrQuery: string[] | number[] | TQuery,
```

Comment:
Missed this in my own suggestion, the query type here needs to bed different from TQuery as this query does not affect the output shape of the function.
```suggestion
		keysOrQuery: string[] | number[] | Query<Schema, Schema[Collection]>,
```

Reply:
fixed